GUI/MainWindow.xaml.cs

```diff
@@ -689,6 +697,58 @@ private void LoadTaskButton_Click(object sender, RoutedEventArgs e)
             UpdateTaskGuiStuff();
         }
 
+        private void ChangeDefautParams_click(object sender, RoutedEventArgs e)
+        {
+            var dialog = new FileSpecificParamWindow(SelectedRawFiles);
+            if (dialog.ShowDialog() == true)
+            {
+                string fullPathofToml = "";
+                int i = 0;
+                foreach (var settings in dialog.FileSpecificSettingsList)
+                {
+                    string directory = Directory.GetParent(SelectedRawFiles[i].FilePath).ToString();
+                    string fileName = Path.GetFileNameWithoutExtension(SelectedRawFiles[i].FileName);
+                    fullPathofToml = Path.Combine(directory, fileName);
+                    Toml.WriteFile(settings, fullPathofToml + ".toml", MetaMorpheusTask.tomlConfig);
+                    i++;
+                }
+
+                foreach (var file in rawDataObservableCollection)
+                {
+                    for (int j = 0; j < SelectedRawFiles.Count(); j++)
+                    {
+                        if (file.FileName.Equals(SelectedRawFiles[j].FileName))
+                        {
+                            file.Parameters = File.ReadAllText(fullPathofToml + ".toml");
```

Comment:
Have this line (reading in file specific params from toml) not only after your GUI dialog result, but also when just adding raw files.